apps/yapms/src/routes/+page.server.ts

```diff
@@ -1 +1,30 @@
+import type { PageServerLoad } from './$types';
+import { globSync } from 'glob';
+import fs from 'fs';
+
+export const load: PageServerLoad = () => {
+	const files = globSync('./src/lib/assets/maps/**/*.svg');
+
+	const search = [];
+
+	for (const file of files) {
+		const data = fs.readFileSync(file, 'utf8');
```

Comment:
i get an error during build related to using 'fs' here. 

"Failed to resolve entry for package "fs". The package may have incorrect main/module/exports specified in its package.json"

Reply:
What command are you using to build?

Reply:
npm run build

Reply:
I'm gonna guess this had something to do with being on windows vs linux.